app/locale/az.coffee

```diff
@@ -952,9 +952,6 @@ module.exports = nativeDescription: "azərbaycan dili", englishDescription: "Aze
 #    robin_title: "Senior Product Manager"
 #    nolan_title: "Sales Manager"
 #    lisa_title: "Business Development Manager"
-#    sean_title: "Territory Manager"
-#    liz_title: "Territory Manager"
-#    jane_title: "Customer Success Manager"
```

Comment:
Similar to the above comment, requires commented out english version of titles: csm/territory_manager to be available to translate.

Reply:
See above comment